sections/main-product.liquid

```diff
@@ -112,6 +112,16 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+            {%- when 'sku' -%}
+              {%- assign current_variant = product.selected_or_first_available_variant -%}
+              <p
+                class="product__sku no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}{% if current_variant.sku.size == 0 %} visibility-hidden{% endif %}"
+                id="Sku-{{ section.id }}"
+                role="status"
+                {{ block.shopify_attributes }}
+              >
+                {{- current_variant.sku -}}
```

Comment:
I think we should have a visually hidden `SKU: ` preceding the code or assign an `aria-label="SKU: {{ current_variant.sku }}"` to this element.

Usually, a SKU is an eight-digits code and simply announcing that set of alphanumeric characters without any context doesn't seem like the right way to do it.

Reply:
I was thinking about that too, otherwise it may not be clear what exactly that number/text is.